### PR TITLE
Use bower's sinon.js release directly from URL,

### PR DESCRIPTION
**Pull request by tomalec**
as the bower repo is outdated.

This should make updates easier. However, it still uses the version as before.

I'd love to bump the Sinon to the latest version (1.17.1-> 4.1.1), but I separated that to reduce the risk of migration, and increase the chance for merging.

 - [ ] CHANGELOG.md has been updated

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "lodash": "^3.7.0",
     "mocha": "^3.1.2",
     "sinon-chai": "^2.7.0",
-    "sinonjs": "^1.14.1",
+    "sinonjs": "http://sinonjs.org/releases/sinon-1.17.1.js",
```
Comment (mroderick) [inline bower.json]: We (the Sinon maintainers) would much prefer if you would use the releases from the CDN, than from the documentation site (which is graciously hosted by GitHub).

```
https://cdn.jsdelivr.net/npm/sinon@1.17.1/pkg/sinon.js
```

**Comment by justinfagnani** [comment]
@usergenic I think you most recently dealt with Sinon versions, does this look right to you?

**Comment by tomalec** [comment]
@usergenic , @justinfagnani any chance to get it there?

**Comment by aomarks** [comment]
Closing this pull request because this repo is now archived. If this PR is still active, please submit a new one at https://github.com/Polymer/tools.

**Pull request closed without merging**